src/utils/utils.ts

```diff
@@ -92,3 +92,32 @@ export const generateForgotPasswordToken = (
   });
   return token;
 };
+
+export const getAllBatches = () => {
```

Comment:
suggestion : we can have this array as global variable ?

Reply:
does not make any difference.